src/useTimeout/useTimeout.stories.ts

```diff
@@ -0,0 +1,90 @@
+/* eslint-disable unicorn/prevent-abbreviations,import/no-extraneous-dependencies */
+import { bind, computed, defineComponent, propType, ref } from '@muban/muban';
+import type { Story } from '@muban/storybook/types-6-0';
+import { html } from '@muban/template';
+import { useTimeout } from './useTimeout';
+import { useStorybookLog } from '../hooks/useStorybookLog';
+
+export default {
+  title: 'useTimeout',
+};
+
+type DemoStoryProps = { startImmediate?: boolean; duration?: number };
+
+export const Demo: Story<DemoStoryProps> = () => ({
+  component: defineComponent({
+    name: 'story',
+    props: {
+      startImmediate: propType.boolean.defaultValue(false),
+      duration: propType.number,
+    },
+    refs: {
+      label: 'label',
+      startButton: 'start-button',
+      cancelButton: 'cancel-button',
+    },
+    setup({ refs, props }) {
+      const [logBinding, log] = useStorybookLog(refs.label);
+      const isTimeoutRunning = ref(false);
+
+      const { startTimeout, cancelTimeout } = useTimeout(
+        onTimeoutComplete,
+        props.duration,
+        props.startImmediate,
+      );
+
+      function onTimeoutComplete() {
```

Comment:
It's hoisted so it doesn't matter for the JS runtime, but this function should be defined above the `useTimeout` imo.